src/lib.rs

```diff
@@ -293,12 +294,67 @@ impl Frame {
         Ok(())
     }
 
+    /// Get the module's name by walking /proc/self/maps
+    #[cfg(all(
+        feature = "print-addresses",
+        unix,
+        not(any(target_os = "macos", target_os = "ios"))
+    ))]
+    fn module_info(&self) -> Result<(String, usize), String> {
```

Comment:
Since the `Result` is used in `print` as if it was an `Option` (with the error string always being discarded), would it make sense to instead return `Option` here?

Reply:
Makes sense. Aplpied this.